about.css

```diff
@@ -0,0 +1,393 @@
+}
+
+.mobile-nav-campaign {
+  color: white;
+  background-color: #ec5242;
+  font-weight: 600;
+}
+
+/* ----------NAV - HAMBURGUER - FINISH------------- */
+
+.about-section {
+  padding: 90px 20px;
+  background-image: url(./assets/icons/about-background.svg);
+  background-repeat: no-repeat;
+  background-position: left top;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 7px;
+  background-color: #f7f7f7;
+}
+
+.nav-btn {
+  position: fixed;
+  top: 20px;
+  left: 20px;
+}
+
+.about-hello {
+  color: var(--orange);
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.about-title {
+  width: 100%;
+  text-align: center;
+  color: var(--orange);
+  font-size: 27px;
+  letter-spacing: 1px;
+  font-weight: 900;
+  margin-bottom: 25px;
+  max-width: 900px;
+}
+
+.about-description {
+  color: var(--dark);
+  text-align: center;
+  background-color: white;
+  border: 2px solid #e2e1e1;
+  padding: 10px;
+  font-size: 12px;
+  line-height: 150%;
+}
+
+.contact-text {
+  color: var(--dark);
+  text-align: center;
+  font-size: 12px;
+  line-height: 150%;
+  font-weight: 600;
+}
+
+.email {
+  border-bottom: 1px solid var(--dark);
+  color: var(--dark);
+  font-weight: 600;
+  font-size: 12px;
+}
+
+.logo-section {
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  border-bottom: 1px solid var(--grey);
+}
+
+.section-title {
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.logo-description {
+  font-weight: 400;
+  font-size: 12px;
+  text-align: center;
+  color: #5f5f5f;
+}
+
+.img-container {
+  border: 1px solid var(--grey);
+  width: 100%;
+  padding: 40px 20px;
+  display: flex;
+  place-content: center;
+  margin: 30px 0;
+}
+
+.logo-img {
+  width: auto;
+  height: 100%;
+  max-height: 50px;
+}
+
+.history-section {
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.nyork {
+  background-image: url(./assets/img/new-york-efect.png);
+}
+
+.tokyo {
+  background-image: url(./assets/img/tokyo-effect-2.png);
+}
+
+.history-container {
+  width: 100%;
+  max-width: 900px;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: center;
+  gap: 10px;
+  margin-top: 20px;
+}
+
+.history-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: calc(100% - 20px);
+  padding: 70px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  cursor: pointer;
+  max-width: 400px;
+  margin: auto;
+}
+
+.card-year {
+  color: white;
+  font-size: 25px;
+  font-weight: 800;
+}
+
+.card-desc {
+  color: white;
+  font-size: 13px;
+  font-weight: 400;
+  text-align: center;
+}
+
+.white {
+  color: white;
+}
+
+.footer {
+  margin-top: 50px;
+  padding: 30px 0;
+  background-color: #242425;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+}
+
```

Comment:
- [ ] For the mobile view of the About Page, the `background-color` of the Footer is `#fff` while the text `color` is `#272a31` for the copyright section. Kindly implement this change for **_only_** the mobile view of the `About` section. 👍 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/233150600-3963de04-9bc9-4ca0-995f-b81aebf5e932.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/233150391-fda02a1e-7e5b-4287-a1cb-8a47070169a5.png)